bundle/render/render_text_output.go

```diff
@@ -26,15 +49,14 @@ var renderFuncMap = template.FuncMap{
 	"italic": func(format string, a ...interface{}) string {
 		return color.New(color.Italic).Sprintf(format, a...)
 	},
+	"printLocations": printLocations,
 }
 
 const errorTemplate = `{{ "Error" | red }}: {{ .Summary }}
 {{- if .Path.String }}
   {{ "at " }}{{ .Path.String | green }}
 {{- end }}
-{{- if .Location.File }}
-  {{ "in " }}{{ .Location.String | cyan }}
-{{- end }}
```

Comment:
I'd rather keep this in the template. The function performs template-like functionality and it can be inlined.

Stab at it (not sure if 100% correct wrt newlines and spacing):
```
{{- range $index, $element := .Locations }}
  {{ if $index eq 0 }}in {{ end}}{{ $element.String | cyan }}
{{- end }}
```

Reply:
It's small enough that you can copy-paste this below.

Reply:
Done